notebooks/Load_Data_and_EDA.ipynb

```diff
@@ -1373,8 +1538,45 @@
    "id": "17aa63ee-0e26-4bcf-82cd-740d6493ea65",
    "metadata": {},
    "source": [
-    "The model performs very well on the test set, with an accuracy and F2-Score of over 0.99."
+    "The prediction model performed quite well on test data, with a final overall accuracy of 0.99 and F2 (beta = 2) score of 0.99. The model only makes 40 mistakes out of 12214 test samples. 17 mistakes were predicting a poisonous mushroom as eatible (false negative), while 23 mistakes where predicting a eatible mushroom as poisonous (false positive). The model’s performance is promising for implementation, as false negatives represent potential safety risks and these errors could lead to consuming poisonous mushrooms, it is minimized to protect users. On the other hand, false positives are less harmful, they may lead to discarding safe mushrooms unnecessarily but does not endanger safety."
```

Comment:
"**_eatible_**" → "**_edible_**."
"**_where_** predicting a **_eatible_** mushroom as poisonous" → "**_were_** predicting an **_edible_** mushroom as poisonous."
"**_does_** not endanger safety" → "**_do_** not endanger safety."